docs/how-to/contribute-code.rst

```diff
@@ -82,10 +82,14 @@ Next, install any additional dependencies for your operating system:
 
     No additional dependencies
 
-  .. group-tab:: Linux
+  .. group-tab:: Unix-like
 
     .. code-block:: bash
 
+      # Freebsd13
+      (venv) $ sudo pkg update
+      (venv) $ sudo pkg install gtk3 gobject-introspection cairo webkit2-gtk3
```

Comment:
1. `FreeBSD`, not `Freebsd`
2. Is there any reason to believe this set of instructions won't also be true for FreeBSD12 and 14? Could this just be `FreeBSD`, not `FreeBSD 13`
3. From a UX perspective, FreeBSD should be *last* in this list of instructions, not first. 
4. This list should also include anything needed to install a version of Python that can be used for development purposes.

Reply:
I'm not I see sure you your previous comments address 2 and 3. Your other commits introduce compatibility with FreeBSD 12/14, but these docs still explicitly say 13 (point 2); and they're still at the *top* of the list of Unix options (point 3).

Reply:
Sorry, I dropped the comments for 2 and 3, however the next commit will address them.